hugo/content/docs/getting_started.md

```diff
@@ -192,17 +192,17 @@ The configuration of our Sprotty application is done via Dependency Injection us
 
 ```typescript
 import { Container, ContainerModule } from 'inversify';
-import { configureModelElement, configureViewerOptions, loadDefaultModules, LocalModelSource, PolylineEdgeView, SEdgeImpl, SGraphImpl, SGraphView, SNodeImpl, TYPES } from 'sprotty';
+import { configureModelElement, configureViewerOptions, loadDefaultModules, LocalModelSource, PolylineEdgeView, SEdge, SGraph, SGraphView, SNode, TYPES } from 'sprotty';
 import { TaskNodeView } from './views';
 
 export const createContainer = (containerId: string) => {
     const myModule = new ContainerModule((bind, unbind, isBound, rebind) => {
         bind(TYPES.ModelSource).to(LocalModelSource).inSingletonScope();
 
         const context = { bind, unbind, isBound, rebind };
-        configureModelElement(context, 'graph', SGraphImpl, SGraphView);
-        configureModelElement(context, 'task', SNodeImpl, TaskNodeView);
-        configureModelElement(context, 'edge', SEdgeImpl, PolylineEdgeView);
+        configureModelElement(context, 'graph', SGraph, SGraphView);
+        configureModelElement(context, 'task', SNode, TaskNodeView);
+        configureModelElement(context, 'edge', SEdge, PolylineEdgeView);
```

Comment:
Here we actually need the classes, not the interfaces. `configureModelElement` defines that model element classes should be used for the _internal model_ used in the command stack and viewer.